### PR TITLE
Batch changes produced by YEdit#save() and #saveAs()

### PR DESCRIPTION
**Pull request by tfesenko**
We have multiple error marker changes are created by the YEdit default implementation for `org.dadacoalition.yedit.editor.YEdit.doSave()` calling `org.dadacoalition.yedit.editor.YEdit.markErrors()`

See https://github.com/ModelSolv/RepreZen/pull/1009#issuecomment-234627391 for details.

### CODE REVIEW

**Comment by tfesenko** [comment]
@andylowry , can you please review?

File: com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerEditor.java [hunk 1]
```diff
@@ -281,21 +285,64 @@ public void updateFoldingStructure(List<Position> positions) {
...
+                        @Override
+                        public void run() {
+                            SwaggerEditor.super.doSaveAs();
+                        }
+                    });
+                }
+                return Status.OK_STATUS;
+            }
+        }.schedule();
         validate();
```
Comment (andylowry) [inline com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerEditor.java]: @tfesenko Did you mean for this validate call to be outside the workspace job?

Comment (tfesenko) [inline com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerEditor.java]: Oops. 
Thanks, @andylowry , fixed it.

**Comment by andylowry** [comment]
@tfesenko Code review passed

**Pull request merged**